recipes/r-beyondcell/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = '1.3.3' %}
+{% set name = "beyondcell" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-beyondcell
+  version: {{ version }}
+
+source:
+   url: https://gitlab.com/bu_cnio/beyondcell/-/archive/v1.3.3/beyondcell-v1.3.3.tar.gz
+   git_tag: v{{ version }}
+   sha256: 8de363a2774913253e85d4efc5c016b9a69d5e245f5730f46116e39a0139dc85
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+
+requirements:
+  build:
+    - {{posix}}filesystem        # [win]
+    - {{posix}}git
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base >=4.0.0
+    - r-bnstruct >=1.0.6
+    - r-cowplot >=1.1.0
+    - r-gdata >=2.18.0
+    - r-ggplot2 >=3.3.2
+    - r-ggrepel >=0.8.2
+    - r-patchwork >=1.0.1
+    - r-plyr >=1.8.6
+    - bioconductor-qusage >=2.22.0
+    - r-rcolorbrewer >=1.1-2
+    - r-scales >=1.1.1
+    - r-see
+    - r-Seurat >=3.2.2
+    - r-useful 
+    - r-viridis >=0.5.1
+    - python=3.6
+    - numba=0.46
+    - umap-learn =0.4.6
+
+  run:
+    - r-base >=4.0.0
+    - r-bnstruct >=1.0.6
+    - r-cowplot >=1.1.0
+    - r-gdata >=2.18.0
+    - r-ggplot2 >=3.3.2
+    - r-ggrepel >=0.8.2
+    - r-patchwork >=1.0.1
+    - r-plyr >=1.8.6
+    - bioconductor-qusage >=2.22.0
+    - r-rcolorbrewer >=1.1-2
+    - r-scales >=1.1.1
+    - r-see
+    - r-Seurat >=3.2.2
+    - r-useful 
+    - r-viridis >=0.5.1
+    - python=3.6
```

Comment:
Does this really need exactly python 3.6 and nothing more recent?

Reply:
Yes, because it requires umap-learn for clustering with Seurat. With python > 3.6 it breaks.